packages/jmix-react-web/src/ui/Tabs/Content.tsx

```diff
@@ -51,3 +51,7 @@ setTimeout(() => {
   checkItems(menuItems);
 }, 1);
 
+window.addEventListener('popstate', () => {
+  checkItems(menuItems)
+});
+
```

Comment:
Fixes for #588 and #548 look straightforward and isolated, let's extact them into separate PRs, so that they don't need to wait for resolution of other review comments.